.github/workflows/csharp.yml

```diff
@@ -1,28 +1,31 @@
-
+---
 name: Csharp CI with .NET
 
 on:
   push:
-    branches: [ master ]
+    branches: [master]
   pull_request:
-    branches: [ master ]
+    branches: [master]
 
 jobs:
-  lint_markdown_files:
+  lint_code_base:
     runs-on: ubuntu-latest
+    name: Lint Code Base
     steps:
-    - uses: actions/checkout@v3
-    - name: Set up Ruby
-      uses: ruby/setup-ruby@v1
-      with:
-        ruby-version: '2.6'
-        bundler-cache: true # runs 'bundle install' and caches installed gems automatically
-    - name: Install gem
-      run: |
-        gem install awesome_bot
-    - name: Run tests
-      run: find . -type f -name '*.md' -exec awesome_bot {} \;
-  
+      - name: Checkout Code
+        uses: actions/checkout@v3
+        with:
+          # Full git history is needed to get a proper list of changed files
+          fetch-depth: 0
+      - name: Lint Code Base
+        uses: github/super-linter@v4
+        env:
+          VALIDATE_ALL_CODEBASE: false
+          DEFAULT_BRANCH: master
```

Comment:
does this mean it only checks for master branch?

Reply:
yes It will compare and check linting of only the files which were modified and compares with master.